brave/config.py

```diff
@@ -174,6 +179,17 @@
     'zenparsing': 'UU7NY6Q9L',
 }
 
+if (os.getenv('GITHUB_SLACK_MAP_FILE', '')):
+    try:
+        filename = os.getenv('GITHUB_SLACK_MAP_FILE')
+        with open(filename, "r") as fh:
+            dynamic_github_slack_mapping = json.load(fh)
+        if dynamic_github_slack_mapping:
+            github_slack_map = {**dynamic_github_slack_mapping, **github_slack_map}
```

Comment:
Why concatenate the two dictionaries, instead of just using the dynamic one (and possibly falling back on the static one if the dynamic one is not available)?

Reply:
@wknapik It's not precisely concat. I'm merging both, the dynamic one if available is loaded, then the static one is allowed to override any keys. I think this has multiple benefits:
* Supports offline operation outside of Jenkins. In whatever way it is used today, we try not to break this
* Allows for manual overrides of some mappings which can prove useful in some cases